frontend/pages/login.tsx

```diff
@@ -67,59 +63,75 @@ const Login = (props: Props) => {
-        helperText={formik.touched.password && formik.errors.password}
-        InputProps={{
-          startAdornment: (
-            <InputAdornment position="start">
-              <PasswordIcon />
-            </InputAdornment>
-          ),
-        }}
-      ></TextField>
+    <Card sx={{ minWidth: 250 }}>
       <Grid container alignItems="center" justifyContent="center" direction="column">
-      <Button variant="contained" size="large" type="submit">
-        Login
-      </Button>
-      <br></br>
+        <Grid container alignItems="center" justifyContent="center" direction="column">
+          <form onSubmit={formik.handleSubmit}>
+            {LoggedInErrored && <Alert severity="error">{LoggedInErrored}</Alert>}
+            <TextField
+              sx={{ marginTop: MARGIN, marginBottom: MARGIN }}
+              variant="outlined"
+              fullWidth
+              label="Email"
+              id="email"
+              value={formik.values.email}
+              onChange={formik.handleChange}
+              error={formik.touched.email && Boolean(formik.errors.email)}
+              helperText={formik.touched.email && formik.errors.email}
+              InputProps={{
+                startAdornment: (
+                  <InputAdornment position="start">
+                    <EmailIcon />
+                  </InputAdornment>
+                ),
+              }}
+            ></TextField>
+            <TextField
+              variant="outlined"
+              sx={{ marginBottom: MARGIN }}
+              fullWidth
+              label="Password"
+              id="password"
+              type={'password'}
+              value={formik.values.password}
+              onChange={formik.handleChange}
+              error={formik.touched.password && Boolean(formik.errors.password)}
+              helperText={formik.touched.password && formik.errors.password}
+              InputProps={{
+                startAdornment: (
+                  <InputAdornment position="start">
+                    <PasswordIcon />
+                  </InputAdornment>
+                ),
+              }}
+            ></TextField>
+            <Grid container alignItems="center" justifyContent="center" direction="column">
+              <Button variant="contained" size="large" type="submit">
+                Login
+              </Button>
+            </Grid>
+          </form>
+        </Grid>
+        <Grid container alignItems="center" justifyContent="center" direction="column">
+          <form
+            method="get"
+            action={`${API.CONFIGS.baseURL}${API.ENDPOINT.AUTHENTICATION.GOOGLE_LOGIN}`}
+          >
+            <Button
+              startIcon={<Google />}
+              variant="contained"
+              size="large"
+              type="submit"
+              sx={{ marginTop: MARGIN, marginBottom: MARGIN }}
+            >
+              Login with Google
+            </Button>
+          </form>
```

Comment:
i was actually thinking that the button is an `href` using an anchor tag. What do you think @MouseAndKeyboard ?